File: docs/Student.java
// give a java student class

public class Student {

    private String name;
    private int age;
    private String studentId;

    // constructor
    public Student(String name, int age, String studentId) {
        this.name = name;
        this.age = age;
        this.studentId = studentId;
    }

    // getters
    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public String getStudentId() {
        return studentId;
    }

    // setters
    public void setName(String name) {
        this.name = name;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setStudentId(String studentId) {
        this.studentId = studentId;
    }
}

File: student.cpp
// give me a student class with getter setter and ctor
#include <string>
#include <iostream>
using namespace std;

class Student {
private:
    std::string name;
    int age;

public:
    // Constructor
    Student(std::string name, int age) {
        this->name = name;
        this->age = age;
    }

    // Getter for Name
    std::string getName() const {
        return name;
    }

    // Setter for Name
    void setName(std::string name) {
        this->name = name;
    }

    // Getter for Age
    int getAge() const {
        return age;
    }

    // Setter for Age
    void setAge(int age) {
        this->age = age;
    }
};
